### ISSUE TITLE
Are all the styles from the theme actually correct?

### ISSUE THREAD

**Issue by bertearazvan** [DHI/react-components#141]
Let's take this thread to report styling that we believe is not correct.

`.MuiIconButton-root:hover`:
- There is a background color on hover. Is that really needed?

`.MuiButton-root`:
- height should be `3rem`?
- padding should be really set?

**Comment by rosudhi** [comment]
Should we discuss with designer teams regarding this? current styles comes from bit.dev codes. and i'm agree that there's some rooms for improvement things here.

**Comment by bertearazvan** [comment]
> Should we discuss with designer teams regarding this? current styles comes from bit.dev codes. and i'm agree that there's some rooms for improvement things here.

Yeah, I will have a meeting tomorrow in regards to some typography variants, I will tackle this as well. Meanwhile, feel free to update this issue with what you notice as you use the Theme in your own projects.
